wolfssl/ssl.h

```diff
@@ -1557,6 +1557,11 @@ WOLFSSL_API int wolfSSL_GENERAL_NAME_set_type(WOLFSSL_GENERAL_NAME* name,
         int typ);
 WOLFSSL_API WOLFSSL_GENERAL_NAMES* wolfSSL_GENERAL_NAMES_dup(
                                              WOLFSSL_GENERAL_NAMES* gns);
+WOLFSSL_API int wolfSSL_GENERAL_NAME_set0_othername(WOLFSSL_GENERAL_NAME* gen,
+                                                    WOLFSSL_ASN1_OBJECT* oid,
+                                                    WOLFSSL_ASN1_TYPE* value);
+
+WOLFSSL_API WOLFSSL_STACK* wolfSSL_sk_GENERAL_NAME_new(void *cmp_func);
```

Comment:
Why cmp_func for the parameter name, is it a compare function? Lets also use camel case here.

Reply:
Can't use camel case here.  Following OpenSSL naming conventions.

Reply:
For the name of the parameter in the function signature it should not matter? Mapping from OpenSSL -> wolfSSL should happen in the wolfssl/openssl/* files, does it throw an error or break when changed?

Reply:
Sorry, I thought convention was to use what openSSL used. I will change to `cmpFunc`.